src/sentry/static/sentry/app/routes.jsx

```diff
@@ -829,6 +829,19 @@ function routes() {
         </Route>
       </Route>
 
+      <Redirect from="sentry-apps/" to="integrations/" />
+      <Route name="Sentry Apps" path="sentry-apps/">
```

Comment:
@NisanthanNanthakumar The name here should be `Integrations` even though we want the path to be `sentry-apps/` since we want to make the experience the same for all types of integrations as much as possible.